migrations/sql-migrations/dynamoToSql/Helper.js

```diff
@@ -35,7 +35,7 @@ class Helper {
       name: projectData.name,
       description: projectData.description,
       processing_config: experimentData.processingConfig,
-      samples_order: experimentData.sampleIds,
+      samples_order: JSON.stringify(experimentData.sampleIds),
```

Comment:
Changed samples_order to be of type jsonb array instead of postgres array becuase it has extra functionality needed to reorder the samples.

When using jsonb we need to use stringify jsons that are going to be inserted